Formula/latexml.rb

```diff
@@ -45,6 +45,13 @@ def install
     end
   end
 
+  def caveats
```

Comment:
This caveat won't be allowed.

Reply:
Why? Should I suggest to install LaTeX through Homebrew instead?